### ISSUE TITLE
[a11y] Missing space character in data set properties

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1188]
**Is your feature request related to a problem? Please describe.**
Please add a space after Number of rows: 

![image](https://user-images.githubusercontent.com/37256050/158610033-3cf06cbd-f356-47f6-b810-1d57a9b09c06.png)

**Comment by maxxxxxdlp** [comment]
Fixed
